core/templates/pages/learner-dashboard-page/card-display/card-display.component.spec.ts

```diff
@@ -256,4 +275,46 @@ describe('CardDisplayComponent', () => {
       });
     });
   });
+
+  it('should handle event emitted by content toggle button', () => {
+    component.tabType = 'progress';
+    fixture.detectChanges();
+
+    spyOn(component, 'handleToggleState').and.callThrough();
+    const button = fixture.debugElement.query(
+      By.directive(ContentToggleButtonComponent)
+    ).componentInstance;
+    button.toggle();
+
+    fixture.detectChanges();
+
+    expect(component.handleToggleState).toHaveBeenCalledWith(true);
+    expect(component.toggleState).toBeTrue();
+  });
+
+  it('should return empty string for getVisibility if tabType is not progress', () => {
+    expect(component.getVisibility()).toEqual('');
```

Comment:
Should we initialize the tab type to something other than "progress" here for good measure?

Reply:
Sorry, I'm a bit unclear on this. Since tabType is already initialized to 'home' in beforeEach, should I update it to something different besides 'home' for this test?

Reply:
Ah I missed that, thanks for checking!